ipatests/test_integration/base.py

```diff
@@ -40,6 +40,19 @@ class IntegrationTest:
     domain_level = None
     fips_mode = None
 
+    @classmethod
+    def setup_class(cls):
+        cls.master.run_command(
+            ['dnf', '-y', 'copr', 'enable', 'ckelley/pki']
+        )
+        cls.master.run_command(['dnf', '-y', 'update', 'pki-core'])
+
```

Comment:
@rcritten @ckelleyRH Note for later: if this depends on a current 389DS it might be best to update it here too.
The current images are up-to-date at the moment so that should not be an issue now but it could.

Reply:
While a general solution would be nice, I'm specifically just trying to hack this so we can test both ways.

Unfortunately the host configuration (cls.master, cls.replicas, etc) are done after setup_class() is called.